jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java

```diff
@@ -239,6 +239,43 @@ public JibContainerBuilder addEnvironmentVariable(String name, String value) {
     return this;
   }
 
+  /**
+   * Sets the directories that the container will create as a volume. Volumes allows persistent
+   * data.
+   *
+   * <p>This is similar to <a href="https://docs.docker.com/engine/reference/builder/#volume">{@code
+   * VOLUME} in Dockerfiles</a>.
+   *
+   * @param volumes the list of volumes
```

Comment:
"the list of directories to set as volumes"

Reply:
Changed.